src/sentry/tasks/derive_code_mappings.py

```diff
@@ -87,11 +87,7 @@ def process_error(error: ApiError, extra: dict[str, str]) -> None:
     default_retry_delay=60 * 10,
     max_retries=3,
 )
-def derive_code_mappings(
-    project_id: int,
-    data: NodeData | None = None,  # We will deprecate this
-    event_id: str | None = None,
-) -> None:
+def derive_code_mappings(project_id: int, event_id: str) -> None:
```

Comment:
You need to define this with `**kwargs` so that old tasks that are still in the queue that pass `data` don't break here

Reply:
I addressed it; thanks!